frame/babe/src/lib.rs

```diff
@@ -224,10 +225,20 @@ decl_storage! {
 		/// on block finalization. Querying this storage entry outside of block
 		/// execution context should always yield zero.
 		Lateness get(fn lateness): T::BlockNumber;
+
+		/// The configuration for the current epoch.
+		EpochConfig: BabeEpochConfiguration;
```

Comment:
I understand why you implemented `Default` for this. However, I'm by far no fan of putting polkadot default values in here. 

Sadly there is no "value" is always set mode for the storage. We could make this an `Option`, `expect` with a proof and write some migration code.